xc/common/utils/prjxray_arch_import.py

```diff
@@ -550,17 +731,26 @@ def get_none_tile_prefix(single_xml):
         yield vpr_tile_type, grid_x, grid_y, meta_fun
 
 
-def add_synthetic_tiles(model_xml, complexblocklist_xml, tiles_xml, need_io):
+def add_synthetic_tiles(
```

Comment:
There shouldn't need to be two add_synthetic_tiles tile functions?  add_synthetic_tiles(num_input=1) and add_synthetic_tiles(num_output=1) should exactly the same?